code-samples/dags/airflow-datasets/dataset_consumer.py

```diff
@@ -0,0 +1,28 @@
+import pendulum
+from datetime import datetime
+
+from airflow import DAG, Dataset
+from airflow.decorators import task
+
+INSTRUCTIONS = Dataset("file://localhost/airflow/include/cocktail_instructions.txt")
+INFO = Dataset("file://localhost/airflow/include/cocktail_info.txt")
+
+with DAG(
+    dag_id="datasets_consumer_dag",
+    start_date=pendulum.datetime(2022, 10, 1, tz="UTC"),
+    schedule=[INSTRUCTIONS, INFO],  # Scheduled on the Datasets
```

Comment:
```suggestion
    schedule=[INSTRUCTIONS, INFO],  # Scheduled on both Datasets
```

just a suggestion. Maybe "schedule to run after both datasets have been updated" but that is a bit long for the comment 😅